timeless/__init__.py

```diff
@@ -68,6 +66,13 @@ def register_api(app, view, endpoint, url, pk="id", pk_type="int"):
 
 
 def register_endpoints(app):
+    from timeless.companies import views as companies_views
+    from timeless.auth import views as auth_views
+    from timeless.reservations import views as reservations_views
+    from timeless.restaurants.locations import views as locations_views
+    from timeless.restaurants.floors import views as floors_views
+    from timeless.restaurants.table_shapes import views as table_shapes_views
```

Comment:
@amihaiemil  Because it's needed to import models from views and we have a situation when we import views then views import models and it's happening before actually Flask app is created. You can check `initialize_extensions` function in this file and you will see that we have the same problem there - we do `DB.init_app(app)` and then import all models inside the function.

Reply:
@amihaiemil So, the main idea is to create Flask app and only then import models.  If do imports as it was before - models will be imported in import time before initializing Flask app.